src/scripting/game_lua_kernel.cpp

```diff
@@ -4238,6 +4226,12 @@ game_lua_kernel::game_lua_kernel(game_state & gs, play_controller & pc, reports
 	luaL_setfuncs(L, map_callbacks, 0);
 	lua_pop(L, 1);
 
+	// Add open_help to the GUI module
```

Comment:
Why do this here, instead of moving it to lua_gui2.cpp?

Given the naming of the other gui functions, `show_help` seems more consistent than `open_help`.

Reply:
Both good points.